### DISCUSSION TITLE
Is there a way to use `top` and `left` instead of `transform`

### DISCUSSION THREAD

**Discussion by Wetoria** [bcakmakoglu/vue-flow#1719] [Q&A] [answered]
My project is used fixed to calculate position, `transform` will broke the feature. So, is there a way to use `top` and `left` to instead of `transform`?

**Comment by bcakmakoglu** [comment, marked as answer]
> My project is used fixed to calculate position

Calculate what position? 

> So, is there a way to use top and left to instead of transform?

No that's not possible.

**Reply by Wetoria** [reply]
I used Vue-Flow to implement Whiteboard in a project as a plugin. And the project is used `fixed` to show some element.

As you know, transform will broke the flow, then I get the position issue.

Thanks for your reply, I'll try to find another way.
